bundler/doc/contributing/BUG_TRIAGE.md

```diff
@@ -33,11 +33,12 @@ If you can reproduce an issue, you're well on your way to fixing it. :)
 Everyone is welcome and encouraged to fix any open bug, improve an error message or add documentation. If you have a fix or an improvement to a ticket that you would like to contribute, we have a small guide to help:
 
   1. Discuss the fix on the existing issue. Coordinating with everyone else saves duplicate work and serves as a great way to get suggestions and ideas if you need any.
-  2. Base your commits on the correct branch. Bugfixes for 1.x versions of Bundler should be based on the matching 1-x-stable branch.
+  2. Base your commits on the correct branch. Bugfixes for 2.x versions of Bundler should be based on the matching 2-x-stable branch.
```

Comment:
I think we can remove this point. PRs should be created against the main branch and backports are handled by maintainers.

Reply:
Removed the line in the latest commit.